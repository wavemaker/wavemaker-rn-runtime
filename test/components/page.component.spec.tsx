import React, { ReactNode } from 'react';
import { Text, TouchableOpacity } from 'react-native';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import WmPage from '@wavemaker/rn-runtime/components//page/page.component';

describe('Test Page component', () => {
    test('Check validity of sample component', () => {
      const tree = renderer.create(<WmPage name="test_Page"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });
});