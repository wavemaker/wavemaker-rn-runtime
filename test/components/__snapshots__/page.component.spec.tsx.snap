// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Page component Check validity of sample component 1`] = `
<View
  onMoveShouldSetResponder={[Function]}
  onMoveShouldSetResponderCapture={[Function]}
  onResponderEnd={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderReject={[Function]}
  onResponderRelease={[Function]}
  onResponderStart={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  onStartShouldSetResponderCapture={[Function]}
  style={
    {
      "bottom": 0,
      "flexDirection": "column",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
/>
`;
